### PR TITLE
When uploading two files to cloud code at the same time sometimes it switches the file names

### PR DESCRIPTION
**Pull request by fn-faisal**

### CODE REVIEW

File: src/components/B4ACodeTree/B4ACodeTree.react.js [hunk 1]
```diff
@@ -69,8 +69,8 @@ export default class B4ACodeTree extends React.Component {
   }
 
   async handleFiles(files) {
```
Comment (davimacedo) [inline src/components/B4ACodeTree/B4ACodeTree.react.js:71]: Can you also remove the async form this function?

File: src/components/B4ACodeTree/B4ACodeTree.react.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ export default class B4ACodeTree extends React.Component {
     if (file) {
       let currentTree = '#'
       B4ATreeActions.addFilesOnTree(file, currentTree)
-      await this.setState({ newFile: '' })
+      this.setState({ newFile: '', filesOnTree: file });
```
Comment (davimacedo) [inline src/components/B4ACodeTree/B4ACodeTree.react.js:82]: form this as well?

**Review by davimacedo — APPROVED** [review]

File: src/components/B4ACodeTree/B4ACodeTree.react.js (hunk 1 above)
Comment (fn-faisal) [reply src/components/B4ACodeTree/B4ACodeTree.react.js]: I have removed async from this function.

File: src/components/B4ACodeTree/B4ACodeTree.react.js (hunk 2 above)
Comment (fn-faisal) [reply src/components/B4ACodeTree/B4ACodeTree.react.js:82]: I have removed async from this function aswell.

**Review by davimacedo — APPROVED** [review]

**Pull request merged**